docs/source/diagrams/ReflectometryReductionOne_LowLvl-v1_wkflw.dot

```diff
@@ -55,17 +66,28 @@ $algorithm_style
   rebin           [label="Rebin"]
   scale           [label="Scale"]
   calcResolution  [label="CalculateResolution"]
+  convertDet      [label="Convert Units\n(AlignBins = True)"]
+  convertMon      [label="Convert Units"]
+  groupDet        [label="Group Detectors"]
+  cropDetWS       [label="Crop Detector\nWorkspace"]
+  rebinDet        [label="Rebin Detectors"]
+  cropMonWS       [label="Crop Monitor\nWorkspace"]
+  calcFlatBg      [label="Calculate Flat\nBackground"]
+  rebinToWS       [label="Rebin To\nWorkspace"]
+  sumDirectBeam   [label="Sum Direct\nBeam"]
+  rebinByDirectBeam [label="Rebin By\nDirect Beam"]
+  normalizeByDirectBeam [label="Normalize By\nDirect Beam"]
+  rebinByTransWS  [label="Rebin By Transmission\nWorkspace"]
```

Comment:
The same for `Rebin By Transmission\nWorkspace`, it should be `RebinToWorkspace`. The workspace we are taking as a reference for the rebinning is the input workspace in wavelength (with all normalizations applied), therefore it could be worth adding a `param` item called `InputWorkspace in wavelength` between `Divide (Detectors/Monitors/Unity)` and `FirstTransmissionRun given?`. This can be then linked to `RebinToWorkspace` to indicate that it is the input property to the algorithm.